scripts/simulation/simulation.pm

```diff
@@ -26,7 +26,7 @@ sub run {
       #$cmd = "$cmd -ex \\\"set breakpoint pending on\\\" ";
       #$cmd = "$cmd -ex run --args ";
 
-      $cmd = "$cmd $gams_root/bin/gams_controller --platform unreal-quad -i $i -n $num --loop-time $time --period $period --queue-length 2000000";
+      $cmd = "$cmd $gams_root/bin/gams_controller -i $i -n $num --loop-time $time --period $period --queue-length 2000000";
```

Comment:
After this merge, we should look into this file and make sure the platform is more extensible in this script

Reply:
This change is already merged into the master and it looks generic, since platform argument is being passed from configs, e.g. "vrep-quad", "vrep-uav", "unreal-quad", etc.